ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -101,41 +101,44 @@ private synchronized void resetClient(AppendEntriesRequest request) {
 
   @Override
   protected void runAppenderImpl() throws IOException {
-    boolean shouldAppendLog;
+    boolean installSnapshotRequired;
```

Comment:
Because that code is not used, I guess.

But it's a good question. I think `LogAppender#runAppenderImpl` either should be empty and abstract or implementation of `GrpcLogAppender#runAppanderImpl` should be moved to there (if possible).  (A different, refactoring PR, maybe?)

Reply:
I agree, it would be a good idea to do it in a separate PR. There are many changes in GrpcLogAppender which would be good to have in LogAppender.